l10n_es_partner/tests/test_l10n_es_partner.py

```diff
@@ -28,8 +28,16 @@ def setUpClass(cls):
 
     def test_search_commercial(self):
         partner_obj = self.env["res.partner"]
-        self.assertTrue(partner_obj.name_search("prueba"))
-        self.assertTrue(partner_obj.name_search("comercial"))
+        prueba = partner_obj.name_search("prueba")
```

Comment:
Hazlo por favor en una línea el assert y el name_search en lugar de popular una variable que no sirve y además que no va con nombre en inglés (aplica a todo el test).

Reply:
Lo de cambiar el nombre de acuerdo, lo de ponerlo en variable, es para comprobar después lo siguiente:

`self.assertTrue(prueba not in not_prueba)`

Ya que hago la búsqueda contraria con el operador `!=`. 

¿Cómo lo harías?